admin/nodes/views.py

```diff
@@ -1,39 +1,29 @@
-from datetime import datetime
+from __future__ import unicode_literals
 
-from django.views.generic import ListView
+from django.views.generic import ListView, DeleteView
+from datetime import datetime
```

Comment:
`datetime` is standard library, should be imported in the first section of imports (with but below `__future__`)